### ISSUE TITLE
第六轮评估第三组API 评估情况

### ISSUE THREAD

**Issue by ReganYue** [PaddlePaddle/docs#5143]
### 文档中英文链接

paddle.nn.BatchNorm

paddle.nn.functional.batch_norm

paddle.nn.BatchNorm1D

paddle.nn.BatchNorm2D

paddle.nn.BatchNorm3D

paddle.nn.BCELoss

paddle.nn.functional.binary_cross_entropy

paddle.nn.BCEWithLogitsLoss

paddle.nn.functional.binary_cross_entropy_with_logits

paddle.nn.BeamSearchDecoder

### 主观评价

```
paddle.nn.BatchNorm			示例代码使用numpy
paddle.nn.functional.batch_norm				示例代码使用numpy，中文文档比英文文档少参数use_global_stats (bool|None, optional)、training，中英文API描述与参数描述多处不一致
paddle.nn.BatchNorm1D				示例代码使用numpy，中文文档比英文文档少一个参数use_global_stats (bool|None, optional)，bias_attr (ParamAttr，可选)中文文档少一个类型。
paddle.nn.BatchNorm2D				示例代码使用numpy，中文文档比英文文档少一个参数use_global_stats (bool|None, optional)，bias_attr (ParamAttr，可选)中文文档少一个类型。
paddle.nn.BatchNorm3D				示例代码使用numpy，中文文档比英文文档少一个参数use_global_stats (bool|None, optional)，bias_attr (ParamAttr，可选)中文文档少一个类型。
paddle.nn.BCELoss				英文文档Shape格式错误    使用numpy
paddle.nn.functional.binary_cross_entropy				英文文档Returns格式错误
paddle.nn.BCEWithLogitsLoss				英文文档Shape格式错误  Examples出错
paddle.nn.functional.binary_cross_entropy_with_logits				英文文档Return type  Returns格式错误
paddle.nn.BeamSearchDecoder				英文文档有Return type   没有参数   
```
